### DISCUSSION TITLE
Add the ability to refresh the package list with a button/key press

### DISCUSSION THREAD

**Discussion by majestic53** [stevenbenner/pacfinder#6] [Ideas]
I noticed that if I removed a package while pacfinder is open, the package will continue to be listed in the app. Would it be possible to add a refresh button and/or allow refresh through CTRL+R?

**Comment by stevenbenner** [comment]
Yes. I agree, that would be a useful addition.

A task I was often doing during my package cleanup was running `pacman -D --asdeps [package]` to set the package install reason and verifying the result in PacFinder. Having a button to trigger a refresh would make that process much better.

Currently this program doesn't have any accelerators and mnemonics for keyboard control. That is a project that would need to be done first before adding a keyboard shortcut for refreshing.

Do you think that a refresh button without a keyboard shortcut would be good enough for the short-term?

Also, I think F5 should be a refresh key-bind as well. That's the button that is in my muscle-memory due to web work.

**Reply by majestic53** [reply]
Hi, thanks for the response.

That's similar to my case. I was using the app mainly to track optional/orphaned packages, and remove them if I found any. But removing one package would often lead to other dependent packages becoming removable. So I'd need to relaunch the app to see those.

Yes, I think a refresh button, either on the top bar, or in the drop-down menu would be fine.

As for the keyboard shortcut, F5 also works. I noticed that some other gnome apps use ctrl^r for refresh, such as gparted. I'm not sure which is more standard?

**Comment by stevenbenner** [comment]
Refresh button added with #7. No keybinds yet.

This will be included in the version 1.2 release.
